db/sqlglue.c

```diff
@@ -625,22 +625,22 @@ static int sql_tick(struct sql_thread *thd)
     if ((rc = check_recover_deadlock(clnt)))
         return rc;
 
-    if ((gbl_epoch_time - clnt->last_sent_row_sec) >=
-        gbl_delay_sql_lock_release_sec) {
+    if (clnt->no_transaction == 0) {
```

Comment:
I don't mean to nitpick but `if (!clnt->no_transaction)` seems slightly better here since `no_transaction` is a boolean flag.

Reply:
oh .. yes .. okay .. i read this late (and already merged) but i can do this ..